tests/src/Integration/Action/EntityPropertyFetchTest.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\rules\Integration\Action\EntityPropertyFetchTest.
+ */
+
+namespace Drupal\Tests\rules\Integration\Action;
+
+use Drupal\Tests\rules\Integration\RulesEntityIntegrationTestBase;
+
+/**
+ * @coversDefaultClass \Drupal\rules\Plugin\Action\EntityPropertyFetch
+ * @group rules_action
+ */
+class EntityPropertyFetchTest extends RulesEntityIntegrationTestBase {
+
+  /**
+   * The action to be tested.
+   *
+   * @var \Drupal\rules\Engine\RulesActionInterface
+   */
+  protected $action;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+
+    // Prepare dummy entity manager.
+    $this->entityManager = $this->getMockBuilder('Drupal\Core\Entity\EntityManager')
+      ->setMethods(['getBundleInfo', 'getStorage'])
+      ->setConstructorArgs([
+        $this->namespaces,
+        $this->moduleHandler,
+        $this->cacheBackend,
+        $this->getMock('Drupal\Core\Language\LanguageManagerInterface'),
+        $this->getStringTranslationStub(),
+        $this->getClassResolverStub(),
+        $this->typedDataManager,
+        $this->getMock('Drupal\Core\KeyValueStore\KeyValueStoreInterface'),
+        $this->getMock('Symfony\Component\EventDispatcher\EventDispatcherInterface')
+      ])
+      ->getMock();
```

Comment:
Why do you have to initialize the entity manager again? This is already don in the parent class? Please add a comment.